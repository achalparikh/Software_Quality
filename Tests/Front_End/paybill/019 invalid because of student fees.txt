login
standard 
Student
paybill
00001
TV
500.00
logout
quit