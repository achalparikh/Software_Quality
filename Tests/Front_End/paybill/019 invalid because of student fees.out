Enter user type - "standard" or "admin":
Logging in as standard user - enter name:
You are now logged in as "Student"
COMMAND: paybill - enter account number:
Paying bill from 00001 - enter payee: 
Paying bill to "TV" - enter amount:
ALERT: 5.00 student fee will be added.
ERROR: Maximum amount of 50.00 exceeded. Try again
You are now logged out
